### PR TITLE
[AC 1313] Update Enterprise Monthly priceId to unblock signups in NY and UT

### PR DESCRIPTION
**Pull request by cyprain-okeke**
## Type of change

```
- [X] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Update Enterprise Monthly priceId to unblock signups in NY and UT

## Code changes

* **src/Core/Utilities/StaticStore.cs:** Update the existing StripeSeatPlanId (`2020-enterprise-org-seat-monthly`) to a new priceId `2020-enterprise-seat-monthly`

## Before you submit

- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- If making database changes - make sure you also update Entity Framework queries and/or migrations
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

**Review by MGibson1 — COMMENTED** [review]
How much has this been tested?

For example, this string is used in auto expansion to update subscriptions. I believe this will fail should the subscription not contain the expected price line item at all.

Are we updating all customers to this new price id in Stripe and need to update code to match? Or will people still be on this price going forward? If people are using the price, I think we need to keep it in our static store.

**Comment by eliykat** [comment]
@MGibson1 check out https://bitwarden.atlassian.net/browse/AC-1314 for context and Stripe fun generally.

TL;DR is that all customers will be using the new priceId and we'll be running a script to update existing customers via the Stripe API. However, the server deployment and the completed execution of that script won't happen simultaneously, and we might also have a rollback/failure scenario. So we might have to keep the old priceId around during a transition period, maybe trying the new one but falling back to the old one if that's rejected by Stripe for an existing subscription. Thoughts?

**Review by r-tome — APPROVED** [review]
Thanks for the context @eliykat.
This looks simple enough to me and @cyprain-okeke has already tested this on a dev environment.
